bin/README.md

```diff
@@ -1,19 +1,19 @@
-# ttmscalr - a command-line tool for working with Scalr
+# ttmscalr - Manage Imagine Math infrastructure through this Scalr-based command-line tool
 
 ## Installing + Configuring
 
 * Clone `git@github.com:thinkthroughmath/scalr.git`
-* Add `scalr/bin` dir to your `PATH` as in __Add to PATH__ below
+* Add the `scalr/bin` directory to your `PATH` environment variable (see: __Add to PATH__ section below)
```

Comment:
the "see __Add to PATH__" bit should really be a link to the below section.  I think to link to a section on the current page it is something like...
(see: [Add to Path](#add-to-path))